python/ray/serve/_private/common.py

```diff
@@ -61,6 +61,7 @@ class ReplicaState(str, Enum):
     RECOVERING = "RECOVERING"
     RUNNING = "RUNNING"
     STOPPING = "STOPPING"
+    DRAINING = "DRAINING"
```

Comment:
I find this state name a little bit confusing because it makes me think of the graceful shutdown procedure :/ don't have a better suggestion off the top of my head though.